contracts/DataFeedConsumer.sol

```diff
@@ -1,24 +1,27 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import { IAggregator } from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregator.sol";
+import {IAggregator} from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregator.sol";
+import {IAggregatorRouter} from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregatorRouter.sol";
 
 contract DataFeedConsumer {
     IAggregator internal dataFeed;
+    IAggregatorRouter internal router;
     int256 public answer;
     uint80 public roundId;
 
-    constructor(address aggregatorProxy) {
+    constructor(address aggregatorProxy, address aggregatorRouter) {
         dataFeed = IAggregator(aggregatorProxy);
+        router = IAggregatorRouter(aggregatorRouter);
     }
 
     function getLatestData() public {
         (
             uint80 roundId_,
-            int256 answer_
-            , /* uint startedAt */
-            , /* uint updatedAt */
-            , /* uint80 answeredInRound */
+            int256 answer_ /* uint startedAt */ /* uint updatedAt */ /* uint80 answeredInRound */,
```

Comment:
Can we keep the previous format? 🙏

Reply:
it's automatically formatted from my vs code... I'll turn off format on save option and update it